cmd/config.go

```diff
@@ -54,7 +54,9 @@ func getConfigPath() (string, error) {
 // mustGetConfigPath must get server config path.
 func mustGetConfigPath() string {
 	configPath, err := getConfigPath()
-	fatalIf(err, "Unable to get config path.")
+	if err != nil {
```

Comment:
> You should still log 'errorIf()'

Currently, minio doesn't work if it fails to get config path, and main.go will show an error anyway.